src/pricecypher/rest.py

```diff
@@ -157,27 +158,37 @@ def _retry(self, make_request):
             # Issue the request
             response = make_request()
 
+            if response.status_code != 429:
+                break
+
+            # Try to find Retry After header in response, which specifies the number of seconds we should wait.
+            wait = response.headers.get('Retry-After')
```

Comment:
will the header always have the "Retry-After" section? is it possible it cannot find it and throws an error?